konig-core/src/main/java/io/konig/core/impl/SimpleLocalNameService.java

```diff
@@ -57,6 +58,7 @@
 		DEFAULT.addStaticFields(RDF.class);
 		DEFAULT.addStaticFields(RDFS.class);
 		DEFAULT.addStaticFields(GCP.class);
+		DEFAULT.addStaticFields(OMCS.class);
```

Comment:
The OMCS class does not contain any property definitions or named individuals, so why is it included in the default SimpleLocalNameService?

Reply:
I have added tableReference to OMCS